src/screens/Navigation/NavigationDetails.tsx

```diff
@@ -0,0 +1,73 @@
+import { ChangeEvent, useEffect, useState } from "react";
+import Image from "next/image";
+import { ProductCard } from "../../components/ProductCard/ProductCard";
+import { Product } from "../../components/ProductCard/types";
+import {
+  ContentContainer,
+  NavigationHeader,
+  ProductGrid,
+  SearchFilter,
+  SearchFilterWrapper,
+} from "./NavigationDetails.styled";
+import { INPUT_PLACEHOLDER, NAV_TITLE } from "./constants";
+import { NavHeader } from "../../components/NavHeader/NavHeader";
+
+export type NavigationProps = {
+  products: Product[];
+};
+
+export const NavigationDetails = ({ products }: NavigationProps) => {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [productsList, setProductsList] = useState(products);
+
+  useEffect(() => {
+    const productsToShow = products.filter((product) => {
+      const searchValue = searchTerm.toLowerCase();
+      const productName = product.name ? product.name.toLowerCase() : "";
+
+      return productName.startsWith(searchValue);
+    });
+
+    setProductsList(productsToShow);
+  }, [products, searchTerm]);
+
+  useEffect(() => {
+    setSearchTerm("");
+    setProductsList(products);
+  }, [products]);
+
+  const handleSearch = (event: ChangeEvent<HTMLInputElement>) => {
+    setSearchTerm(event.target.value);
+  };
+
+  return (
+    <>
```

Comment:
you dont need this extra <>

Reply:
When i remove it i get the jsx error

Reply:
OH my this one you actually need LOL 
I thought you had only one component inside 
my bad! 🙈